### ISSUE TITLE
Conv3d and ConvTransposed3d slower on TensorRT8.6.1.6 than TensorRT8.2.0.3 ?

### ISSUE THREAD

**Issue by QingXIA233** [NVIDIA/TensorRT#3269]
## Description

Recently, in order to use the impressive model [**FasterViT**](https://github.com/NVlabs/FasterViT) as the backbone of my occupancy prediction task, I upgraded my TensorRT version from 8.2 to 8.6. FasterViT runs much faster with TensorRT 8.6 than 8.2. Everything is OK. However, when I convert the whole model to TensorRT and measure the GPU latency, **it gots larger latency on TensorRT 8.6**. I measured the latency of each part of my model, which is shown in this table:
![image](https://github.com/NVIDIA/TensorRT/assets/64100948/d06a174c-2555-4421-986e-d788c0085fe1)

As you can see, the neck3d part and head part are slower. These two modules only contain some ops such as: Conv3d, ConvTransposed3d, ReLU, PReLU and trilinear interpolate. The reason why I'm suspicious of the Conv3d and ConvTransposed3d getting slower is that my **neck3d uses a lot of these two operators**. Therefore, I wonder whether upgrading TensorRT has bad influence on some operators or there are other reasons. As for the latency comparison, I used the same onnx model of each part to convert to tensorrt with the same device. **The only difference is the TensorRT version**. Could you help me cause I'm really confused. Thanks.

## Environment

**TensorRT Version**:
8.6.1.6

**NVIDIA GPU**:
NVIDIA GeForce 3090

**NVIDIA Driver Version**:

**CUDA Version**:
11.4
**CUDNN Version**:

Operating System:
linux

Python Version (if applicable):

Tensorflow Version (if applicable):

PyTorch Version (if applicable):

Baremetal or Container (if so, version):

## Relevant Files

**Model link**:

## Steps To Reproduce

**Commands or scripts**:

**Have you tried [the latest release](https://developer.nvidia.com/tensorrt)?**:

**Can this model run on other frameworks?** For example run ONNX model with ONNXRuntime (`polygraphy run <model.onnx> --onnxrt`):

**Comment by zerollzeng** [comment]
Are you using FP32 or FP16? can you reproduce it with [trtexec](https://github.com/NVIDIA/TensorRT/tree/main/samples/trtexec)? it will print the latency in the summary at the end, and we take it as metric to see if it's perf regression.

If it's FP32, when building the engine, you can try specify `--preview=-disableExternalTacticSourcesForCore0805` in trtexec to see if it solve your problem.

**Comment by ttyio** [comment]
Closing since no activity for more than 3 weeks, pls reopen if you still have question, thanks!
